activities/communication/index.md

```diff
@@ -20,3 +20,16 @@ Phone numbers and lines are managed in our VOIP supplier Voys' [online managemen
 * ['Users'](https://freedom.voys.nl/client/415559/user/): Who can log in
 * ['VoIP accounts'](https://freedom.voys.nl/client/415559/phoneaccount/): Basically lines we can assign
 * ['Voicemails'](https://freedom.voys.nl/client/415559/voicemail/): Voice mail boxes
+
+## Email signature
+
+In order to clearly convey who is communicating, this template for email signature could be used. It has no styling in order to make it more probable that email clients and mailing lists will handle the signature gracefully.
+
+Name, Role  
+Email | Phone | Twitter 
+Foundation for Public Code https://publiccode.net  
+
+*Example:*
+Boris van Hoytema, Chief Executive and secretary
+boris@publiccode.net | +31 20 2 444 555 | @bvhme
+Foundation for Public Code https://publiccode.net  
```

Comment:
```suggestion
[Foundation for Public Code](https://publiccode.net/) 
```
The lack of punctuation here bothers me. I suggest we hyperlink the name.

Reply:
@ElenaFdR  If we were to not use any markup in the footer, what sort of punctuation would make you less bothered?